src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -910,19 +912,21 @@ public function validateLt($attribute, $value, $parameters)
     {
         $this->requireParameterCount(1, $parameters, 'lt');
 
-        $comparedToValue = $this->getValue($parameters[0]);
+        $comparedToValue = $this->getValue($parameters[0] ?? '');
```

Comment:
Remind me why $parameters[0] would be empty / null again?

Reply:
In the [original issue](https://github.com/laravel/framework/issues/28258), the below example was used in the **Steps to Reproduce**:

```
       $canBeNull = null; // assume it is coming from client
        $validator = \Validator::make([
            'min_rate' => null,
            'max_rate' => 100,
        ], [
            'min_rate' => ['required', 'numeric',],
            'max_rate' => ['required', 'numeric', 'gte:'.$canBeNull],
        ]);

        dd($validator->errors()->toArray());
```

It seems kind of contrived. For example, if the field under comparison is provided by the end user, and they didn't enter any data in that field, the [default Middleware](https://github.com/laravel/laravel/blob/master/app/Http/Kernel.php#L20) would convert the empty string to `null`. But in that case `parameters[0]` would be the name of the field as opposed to the raw `null` value provided in the OP's example. But, if the dev set up their code like the OP, in which case `$parameters[0]`would be `null`, that throws off the validation process as described [here](https://github.com/laravel/framework/issues/28258#issuecomment-489324103). That's why I included the null coalescing operator. 

Also, **one thing I noticed in some local testing using my PR** is that if the you add the `integer` validation rule along with the `gte`, `lt`, etc. You could end up with a validation messages that can be redundant if not a little confusing . See below example. If this is not acceptable, I will try to develop a work around. 

```
array:1 [▼
  "validateField" => array:2 [▼
    0 => "The validate field must be an integer."
    1 => "The validate field must be a number."
  ]
]
```

for:

```
	$comparisonField = 123;
	$validator = Validator::make([
		'validateField' => 'string',
	], [
		'validateField' => ['required', 'integer', 'gte:'.$comparisonField],
	]);

	dd($validator->errors()->toArray());
```